aten/src/ATen/nn_parse.py

```diff
@@ -286,19 +286,23 @@ def backward_declaration(base, thnn_functions):
                   if arg['name'] != 'inplace']
     arguments += base['buffers']
 
+    if 'upsample' in base['name']:
```

Comment:
Why did this get moved?

Reply:
No particular reasons now that the new arg is inserted to correct place instead of appended. Previously it was appended after a bunch out output args was added, which made understanding this code difficult for me.

I guess one benefit of moving this up is that now the code is more readable as it first constructs the input arg before adding the output args.